src/lib/flow.js

```diff
@@ -242,6 +243,11 @@ export async function collectFlowCoverageForFile(
   if (parsedData && !parsedData.error) {
     parsedData.filename = filename;
     parsedData.annotation = await genCheckFlowStatus(flowCommandPath, filename);
+
+    if (unflowedFilesNoCoverage && parsedData.annotation === 'no flow') {
+      parsedData.expressions.uncovered_count += parsedData.expressions.covered_count;
```

Comment:
I think this might be better suited as an option that the reporter uses, similar to how the `threshold` is used to set colors in the output. There are a few different annotations to pick from, so for example people might want to consider `flow weak` as being not-good enough.

Reply:
@ryan953 that sounds reasonable to me, at least if "flow weak" is not planned for removal anymore.

Another option that I've been thinking of while looking into this PR is that we could consider to expose this as a option which just exclude the non "flow" annotated files completely, e.g.

```flow-coverage --exclude-non-flow ...```

when `--exclude-non-flow` is specified, we could exclude the files that are missing a flow annotation automatically, as if they were excluded by manually specify a glob to exclude with the `-x` cli option.

On the contrary the tool will keep the current behavior if --exclude-non-flow has not been specified.

 @ryan953 @uforic @dmnd How this approach sounds to you?

@uforic once we agreed on the behavior of the new options, we should also include some new test cases to test the new behavior (so that we can ensure that we don't break it once we apply further changes to the project).

Reply:
@rpl in my project I have a goal of 100% coverage (i.e. 100% of files opted into `@flow`, and each file at 100% coverage). 

I'm using this tool to measure progress. Treating files missing `@flow` as excluded doesn't work because it will hide how much work is remaining. The 0% coverage files won't be included in the denominator.

As an example, check out our historical progress:

![image](https://user-images.githubusercontent.com/4427/36110507-aa577fe4-0fd7-11e8-9948-5ddce7cfa0a7.png)

![image](https://user-images.githubusercontent.com/4427/36110522-b00f73ec-0fd7-11e8-9ce3-29966a4c69bc.png)

Reply:
@dmnd thanks a lot for the detailed comment! it really helped to better understand the use case.

how about a single new cli option called "--strict" (or "--strict-coverage") which makes flow-coverage-report consider completely uncovered any file that is not "stricly @flow" (and so both the non annotated and the "flow weak" ones)?
it should be simple enough to clearly describe what the option does in the cli help message, and it should be able to cover the use case described in @dmnd, as well as help to guide a project that wants to reduce the amount of "flow weak" code.

@uforic @dmnd @ryan953 how that sounds?

Reply:
A "--strict" option sounds great to me @rpl. :+1:

In fact, that's how I expected the report to work in the first place, until I got surprising numbers and dug into the implementation. Given that, I'm of the mind that it should be the default for flow-coverage-report, but I leave that up to you to decide. I personally don't have a good understanding of the nonstrict use case.

Reply:
@dmnd @uforic ok, let's do it! I've opened a new #141 which included this changes (with the option renamed to --strict-coverage) and a new test case to explicitly verify the new "strict coverage" mode behaviors.

In the meantime I'm closing this PR in favor of the new one which includes the new test case.